<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1802</width>
    <height>1042</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>速度规划程序</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="qCustomPlot" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1061</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_load_main_path">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>60</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>InputPath</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_plan_speed">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>SpeedPlan</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="profile" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>1771</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="profile_3" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>1771</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="profile_2" native="true">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>0</y>
      <width>611</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1802</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
